pom.xml

```diff
@@ -26,7 +26,7 @@
   <properties>
     <java.version>11</java.version>
     <folio-spring-base.version>1.0.3</folio-spring-base.version>
-    <openapi-generator.version>4.3.1</openapi-generator.version>
+    <openapi-generator.version>5.1.0</openapi-generator.version>
```

Comment:
Major version change. Hope it does not break anything used by `mod-search`.

Reply:
In mod-search there are a lot of integration tests that can verify if service is able to run and perform as expected.
Also, I've checked breaking changes without fallback between 4.3.1 and 5.0 release, they should affect `mod-search`